ia/src/ml_agent.py

```
@@ -121,6 +122,34 @@ def analyze_vision(client):
+    vision_data = analyze_vision(client)
+
+    if vision_data["food"]:
+        get_food(client)
+    else:
+        client["move"]["consecutive_turns"] += 1
+        if client["move"]["consecutive_turns"] >= 3:
+            client["move"]["consecutive_turns"] = 0
+            client["move"]["forward"] = True
+            protocole.execute_command(client, "Forward", None)
+        else:
+            if random.random() < 0.7:
```

Ruff (0.11.9)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)